docker-compose.yml

```
@@ -18,6 +18,20 @@ services:
       volumes:
         - redis-data:/data
 
+    extractor:
+      image: thedeep/deep-extractor:latest
+      build:
+        context: ./deepl-service/deepl-deep-integration/
```

Let's use deepl-service directly instead of cloning it inside that directory.

```suggestion
                context: ./deepl-service/
```